### PR TITLE
fix grammar error: return should be returns.

### PR DESCRIPTION
**Pull request by liangwenguo**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by dixudx** [comment]
@liangwenguo Would you please find other similar grammar errors. So that we could fix them up in a single PR. Thanks.

**Comment by liangwenguo** [comment]
@dixudx OK

**Review by jamiehannaford — APPROVED** [review]
/lgtm

**Comment by liangwenguo** [comment]
@jamiehannaford i find  similar grammar errors in other files.

**Comment by liangwenguo** [comment]
@dixudx PTAL, Thank you!

**Comment by liangwenguo** [comment]
@dixudx PTAL, Thank you!

File: cmd/kubeadm/app/phases/upgrade/postupgrade_v18_19.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
+/*
```
Comment (dixudx) [inline cmd/kubeadm/app/phases/upgrade/postupgrade_v18_19.go]: This file does not belong to this PR.

**Review by dixudx — CHANGES_REQUESTED** [review]

File: cmd/kubeadm/app/phases/upgrade/postupgrade_v18_19.go (hunk 1 above)
Comment (liangwenguo) [reply cmd/kubeadm/app/phases/upgrade/postupgrade_v18_19.go]: the conflict told me that i had deleted the file, i create a new PR.
